environment.rb

```diff
@@ -0,0 +1,98 @@
+module Environment
+  class Depot
+    attr_accessor :packs 
+    
+    def initialize
+      @packs = {
+        simple_transportation_pack: { 
+          intelligence:[:cellphone],
+          tems: [:medipack, :chevy_versa]
+        },
+        standard_transportation_pack: {
+          intelligence: [:cellphone, :antena],
+          arsenal: [:colt_1911],
+          items: [:handcuffs, :medipack, :chemistry, :chevy_versa]
+        },
+        simple_mission_pack: {
+          intelligence: [:infopack, :laptop, :cellphone, :antena],
+          arsenal: [:colt_1911],
+          items: [:handcuffs, :medipack,:chemistry,:finalcial]
+        },
+        standard_mission_pack: {
+          intelligence: [:infopack, :laptop, :cellphone, :antena],
+          arsenal: [:remington_870, :colt_1911, :machete, :hatchet],
+          items: [:handcuffs, :medipack, :chemistry, :finalcial]
+        } 
+      }
+     end
+
```

Comment:
Extra lines are taken in Ruby as bad practice. The [Rubocop](https://docs.rubocop.org/rubocop/1.23/index.html) gem can guide you through this. Try to install it when you have the time, it will help you to improve this.

Reply:
Thanks!